config/schedule.rb

```diff
@@ -35,10 +35,6 @@ def local(time)
   runner "Reports::RegionCacheWarmer.call"
 end
 
-every :month, at: local("04:00 am"), roles: [:seed_data] do
-  rake "db:purge_and_reseed"
-end
```

Comment:
I'd say keep this. Keeping the data current (seeing sensible data for the most recent month) is important.

Reply:
The problem is it will blow away all the data and then regen, including any changes or tweaks made manually or any manual user accounts.  I think we can remove this and work on https://app.clubhouse.io/simpledotorg/story/2496/create-seed-data-since-the-last-seed-job-ran over the next month or so to get updated data functionality.

WDYT?

Reply:
That is ideal, I agree, as long as we can do that within the month 😄  Actually, yeah, let's keep it off. We can re-run manually if we don't get to ch2496 before March 1, and the data starts getting stale.